addons/sourcemod/scripting/ReservedSlot.sp

```diff
@@ -222,5 +248,5 @@ stock bool KickValidClient(const char[] sName, const char[] sSteam32ID, AdminId
 
 stock void ExecuteKickValidClient(int client, const char[] sName, const char[] sSteam32ID, AdminId admin, int Immunity)
 {
-	KickClientEx(client, "Kicked for reserved slot (%s joined).", sName);
+	KickClientEx(client, "Kicked for reserved slot. (%s joined)", sName);
```

Comment:
wow didnt know about this punctuation thing so according to this:
- https://www.yourdictionary.com/articles/punctuation-inside-outside-parentheses

it should be
```
Kicked for reserved slot. (%s joined).
```